Ryujinx.HLE/HOS/Services/Time/TimeZone/Types/TimeZoneRule.cs

```diff
@@ -22,17 +24,32 @@ struct TimeZoneRule
         [MarshalAs(UnmanagedType.I1)]
         public bool GoAhead;
 
-        [MarshalAs(UnmanagedType.ByValArray, SizeConst = TzMaxTimes)]
-        public long[] Ats;
+        [StructLayout(LayoutKind.Sequential, Size = sizeof(long) * TzMaxTimes)]
+        private struct AtsStorageStruct {}
```

Comment:
```suggestion
        private struct AtsStorageStruct { }
```
Sorry, I missed this one on last review.

Reply:
This and the other comments were not addressed (or maybe they were and somehow got reverted with the rebase?)

Reply:
huh odd? I was pretty sure I did address that one

Reply:
If you applied the suggestion from GitHub, then I guess you forgot to pull the changes on your local branch and they got lost on rebase.